target/linux/qualcommax/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -44,6 +44,11 @@ platform_pre_upgrade() {
 platform_do_upgrade() {
 	case "$(board_name)" in
 	arcadyan,aw1000|\
+	askey,rt5010w-d187)
```

Comment:
this shouldnt be here since you are changing the platform upgrade of another device

Reply:
move this to the the prpl,haze section, no need to create duplicate of the same command

Reply:
fixed